salt/pillar/git_pillar.py

```diff
@@ -319,4 +314,4 @@ def ext_pillar(minion_id,
 
     pil = Pillar(opts, __grains__, minion_id, branch)
 
-    return pil.compile_pillar()
+    return pil.compile_pillar(ext=False)
```

Comment:
This change alone is causing these [test](https://jenkins.saltstack.com/job/salt-pr-rs-cent7-n/2954/testReport/junit/integration.modules.pillar/PillarModuleTest/test_ext_pillar_env_mapping/) [failures](https://jenkins.saltstack.com/job/salt-pr-rs-ubuntu14.04-n/2954/testReport/junit/integration.modules.pillar/PillarModuleTest/test_ext_pillar_env_mapping/).  I'm not sure what is going on here.  Perhaps @rallytime or @cachedout know more than I do.